src/AppBundle/Repository/EventRepository.php

```diff
@@ -396,6 +396,135 @@ private function getUsernamesSql(Event $event): string
         return ltrim(implode(',', $usernames), ',');
     }
 
+    /**
+     * @param Event $event
+     * @return int[]
+     */
+    public function getBytesChanged(Event $event): array
+    {
+        $firstLast = $this->getFirstLastRevisions($event);
+
+        $result = [];
+        foreach ($firstLast as $name => $pages) {
+            $wiki = $event->getWikiByDomain($name);
+
+            /** @var EventWikiRepository $eventWikiRepo */
+            $eventWikiRepo = $this->em->getRepository('Model:EventWiki');
+            $eventWikiRepo->setContainer($this->container);
+
+            $dbName = $eventWikiRepo->getDbNameFromDomain($wiki->getDomain());
+
+            $first = [];
+            $last = [];
+            foreach ($pages as $info) {
+                $first[] = (int)$info['first']['id'];
+                $last[] = (int)$info['last']['id'];
+            }
+
+            $result[$name] = $this->getBytesAfterEvent($dbName, $last)
+                - $this->getBytesBeforeEvent($dbName, $first);
+        }
+
+        return $result;
+    }
+
+    /**
+     * Returns an SQL-friendly stringified list of identifiers
+     *
+     * @param int[] $ids
+     * @return string
+     */
+    private function idList(array $ids): string
```

Comment:
This is nice! I would even put it in `Repository`, as I think we could it use it in other places

Reply:
Done.